index.html

```diff
@@ -4,19 +4,28 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
It's not that critical for us, because we have a small project, but this line tells the browser, that the user will need these resources from another origin, so it allows quicker to connect needed recourses to the code and realise them in realtime.